index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--Header Section (Toolbar)-->
+    <header class="header backgrnd-clr">
+      <nav class="navbar">
+        <img src="images/icons/Welcome.png" alt="logo" />
+        <img src="images/icons/Union.png" alt="hamburger-menu" />
+      </nav>
+    </header>
+    <!--Main content-->
+    <main class="main-content">
+      <!--Headline Section-->
+      <article class="head-line backgrnd-clr">
+        <!--Greeting text-->
+        <h1 class="greeting greeting-text headline-box">
+          Hey there. I’m <br />
+          Mariane
+          <span class="text-color">I’m a software developer</span>
```

Comment:
The subTitle should use hx tags (h2...h6). I recommend you to use `h2` tag for the subTitle.
```html
<h2 class="text-color">I’m a software developer</h2>
```
This way will be semantically correct and the screen readers and other technologies will thank you.